.github/actions/get-configuration/action.yml

```diff
@@ -29,6 +32,7 @@ runs:
         echo "k8s_version=${{ github.event.inputs.k8s_version || '1.30.3' }}" >> $GITHUB_OUTPUT
         echo "istio_version=1.20.3" >> $GITHUB_OUTPUT
         echo "k3d_version=5.7.4" >> $GITHUB_OUTPUT
+        echo "modulectl_version=1.0.0" >> $GITHUB_OUTPUT
```

Comment:
I think we need to document somewhere that we need to bump the modulectl version here when we do a new modulectl release, otherwise it might be forgotten?

Reply:
I guess if we want to use new features of it we need to bump it anyways?

Reply:
yes, but with the Kyma CLI we used the latest version, so it is a new thing to update the modulectl version for the E2E tests

Reply:
Alright. Any proposals where to document this?

Reply:
Is the modulectl release documented somewhere? Maybe we can add it there?

Reply:
I don't we have documents about modulectl yet.